v2/pkg/apis/kubeflow/v2beta1/constants.go

```diff
@@ -21,6 +21,8 @@ const (
 	EnvKubeflowNamespace = "KUBEFLOW_NAMESPACE"
 	// DefaultRestartPolicy is default RestartPolicy for ReplicaSpec.
 	DefaultRestartPolicy = common.RestartPolicyNever
+	// Launcher will fail on start if workers DNS names are not ready so we set OnFailure restart policy
```

Comment:
This comment is part of the godoc. I don't think we need to state the reasons for the change here. Just document what this is.